_pages/business-units/18f/chapters/product.md

```diff
@@ -90,3 +86,11 @@ Here are a few particularly useful essentials:
 - **Google Docs, Drive, and Slides:** We often need to collaborate on documents and slide decks, and we do that through Google's productivity apps.
 - **18F templates:** You can get our document and slide templates through [18F's Visual Identity Guide](https://brand.18f.gov/templates). Please don't make major changes to the base templates; start a conversation in [#18f-branding](https://gsa-tts.slack.com/app_redirect?channel=18f-branding) instead!
 - **Mural:** We use [Mural]({{site.baseurl}}/murally/) as a collaborative whiteboard and sticky note tool. It's a particularly useful tool for brainstorming with distributed teams.
+
+### Suggested reading
```

Comment:
Are you sure about these links? I'm not sure I'd ever seen the pages.18f.gov domain before today, and I wonder if we have fresher handbook content for some of these things.

Reply:
Heard. This is one of many PRs I need to make to update this page, so I will take a look at this section